MapboxCoreNavigation/RouteController.swift

```diff
@@ -277,10 +279,11 @@ open class RouteController: NSObject, Router {
         return false
     }
     
-    internal func userIsWithinRadiusOfRoute(location: CLLocation) -> Bool {
+    internal func userIsWithinEnvelope(of step: RouteStep, at location: CLLocation) -> Bool {
```

Comment:
“Envelope” in a spatial context is a bit novel to this codebase; is there a specific reason for using the term? The implementation still relies on a radius; is that intentional?